src/rules/converters/trailing-comma.ts

```diff
@@ -0,0 +1,120 @@
+import { RuleConverter } from "../converter";
+
+export const convertTrailingComma: RuleConverter = tslintRule => {
+    const eslintArgs =
+        tslintRule.ruleArguments.length !== 0
+            ? collectArguments(tslintRule.ruleArguments)
+            : undefined;
+
+    return {
+        rules: [
+            {
+                ruleName: "comma-dangle",
+                ...(eslintArgs && { ruleArguments: [eslintArgs] }),
+            },
+        ],
+    };
+};
+
+function collectArguments(args: any[]) {
```

Comment:
Mmhh, `any`s.. could you please `type` these out? It's a little scary to me to have so many `any`s outside of the original `tslintRule.ruleArguments`.